cmd/agent/app/import.go

```diff
@@ -128,6 +115,18 @@ func doImport(cmd *cobra.Command, args []string) error {
 		src := filepath.Join(oldConfigDir, "conf.d", f.Name())
 		dst := filepath.Join(newConfigDir, "conf.d", f.Name())
 
+		if f.Name() == "docker_daemon.yaml" {
```

Comment:
We write it in the same directory, but we should avoid copying the old `docker.yaml` configuration file as it could overwrite the one we just created.